### PR TITLE
fix: http methods

### LINKED ISSUE #147: Adjust endpoints that update configurations to use method patch instead of post
The configuration endpoints performing update operations currently use the http method `post`. We think that `patch` fits a little better here.

- [ ] Update these endpoints
- [ ] Update in the frontend

### PR DESCRIPTION
**Pull request by TheRealKasumi**
closes #147 

As discussed it would probably be "more correct" to use `patch` instead of `post` at some points. So let's do it, should be easy.
Note: Frontend must be done before merging.

### CODE REVIEW

File: documentation/api/TesLight API.postman_collection.json [hunk 1]
```diff
@@ -229,7 +230,7 @@
 								{
 									"key": "file",
 									"type": "file",
-									"src": ""
+									"src": "/C:/Users/Kasumi/Desktop/xl/all_the_things_she_said.fseq"
```
Comment (JakobLichterfeld) [inline documentation/api/TesLight API.postman_collection.json:233]: I don't get this insider 😅

**Review by JakobLichterfeld — CHANGES_REQUESTED** [review]
lgtm beside one line, see in-line comment

File: documentation/api/TesLight API.postman_collection.json (hunk 1 above)
Comment (TheRealKasumi) [reply documentation/api/TesLight API.postman_collection.json:233]: Oooopsi xD

**Review by JakobLichterfeld — DISMISSED** [review]
lgtm

**Comment by danielbayerlein** [comment]
@TheRealKasumi I've changed the HTTP request methods in the UI.

**Review by danielbayerlein — DISMISSED** [review]

**Comment by TheRealKasumi** [comment]
Works for me when you are sure the methods are correct😋. I can't really test it right now. But in case of problems we can do a quick fix I guess.

**Comment by TheRealKasumi** [comment]
Seems to be all working. Maybe a small adjustment is required after https://github.com/TheRealKasumi/TesLight/pull/172.

**Review by TheRealKasumi — COMMENTED** [review]
Thanks! From my side we can merge.

**Review by danielbayerlein — APPROVED** [review]

**Pull request merged**